apps/default/Tiltfile

```diff
@@ -0,0 +1,61 @@
+if k8s_context() != "k3d-otel-basic":
+  fail("Expected context to be k3d-otel-basic")
+
+include("../../cluster-infra/cert-manager/Tiltfile")
```

Comment:
These includes are for the ability to do depends on right?

Reply:
@adrielp, you're correct. In this particular scenario the default/basic stack requires Cert-Manager, OpenTelemetry Operator, and Jaeger Operator. When `tilt up` is run against this file, Tilt will merge the current file with the included ones and respect the "resource_deps" of all combined Tiltfiles.